### ISSUE TITLE
Unexpcted `pachctl put-file` behaviour 

### ISSUE THREAD

**Issue by orenyomtov** [pachyderm/pachyderm#1132]
Running:
```
pachctl put-file data master sales -c -f https://raw.githubusercontent.com/pachyderm/pachyderm/v1.2.1/doc/examples/fruit_stand/set1.txt
```

Results in:
```
Oren$ pachctl list-file data master sales/pachyderm/pachyderm/v1.2.1/doc/examples/fruit_stand
NAME                                                                  TYPE                MODIFIED            LAST_COMMIT_MODIFIED   SIZE                
/sales/pachyderm/pachyderm/v1.2.1/doc/examples/fruit_stand/set1.txt   file                3 hours ago         master/0               874 B
```

Version information:
```
pachctl version
COMPONENT           VERSION             
pachctl             1.2.4               
pachd               1.2.4               
```

**Comment by jdoliner** [comment]
This definitely appears to be a bug, I think I may have introduced this recently while fixing another bug. Will have a look.

**Comment by jdoliner** [comment]
Fixed by #1161
